requestOption.js

```diff
@@ -43,6 +43,13 @@ SuiteRequestOption.prototype = {
       .concat([headerToSet]);
   },
 
+  getHeader: function(name) {
+    var result = this.headers
+      .filter(this._exactHeader(name));
```

Comment:
It is not present in node 0.12 , i sticked to not using es6 features, like they are not used in this codebase
If we decide on the minimum version is 4.0 we can start using it